cypress/e2e/delete_totp/index.cy.ts

```diff
@@ -23,21 +10,15 @@ describe("delete TOTP connexion", () => {
 
     cy.contains("L’application d’authentification a bien été supprimée.");
 
-    cy.mailslurp()
-      // use inbox id and a timeout of 30 seconds
-      .then((mailslurp) =>
-        mailslurp.waitForLatestEmail(
-          "eab4ab97-875d-4ec7-bdcc-04323948ee63",
```

Comment:
we should update the email of the users in the sql file, to use a domain like example.com

Reply:
Beau geste